### PR TITLE
Fix Mistral Large 3 nightly test

### PR DESCRIPTION
**Pull request by b8zhong**
`python /sgl-workspace/sglang/test/registered/8-gpu-models/test_mistral_large3.py`

```

============================================================
Mistral-Large-3 Results Summary
Dataset: gsm8k
Baseline: 0.85
============================================================

Model 1: mistralai/Mistral-Large-3-675B-Instruct-2512-NVFP4
  Performance: PASS, output: 3773.9 tok/s
  Accuracy: PASS
  Score: 0.957

============================================================
OVERALL: ALL TESTS PASSED
============================================================

.
----------------------------------------------------------------------
Ran 1 test in 2366.479s

OK
```

---
### CI States

Latest PR Test: :x: **Missing `run-ci` label** — add it to run CI tests.
Latest PR Test (Extra): :x: **Blocked** — `run-ci` is required first.

### CODE REVIEW

File: python/sglang/srt/layers/quantization/compressed_tensors/schemes/compressed_tensors_w4a4_nvfp4_moe.py [hunk 1]
```diff
@@ -311,10 +311,10 @@ def apply_weights(
             router_logits = topk_output.router_logits
             topk_config = topk_output.topk_config
 
-            # Quantize input hidden states using fp4_quantize
+            # global_scale must be shape [1] (strict in cute-dsl backend).
             hs_fp4_bytes, hs_sf_bytes = fp4_quantize(
                 x,
-                layer.w13_input_scale_quant,
+                layer.w13_input_scale_quant[:1],
```
[the remark this answers is not in the document]
Comment (b8zhong) [reply python/sglang/srt/layers/quantization/compressed_tensors/schemes/compressed_tensors_w4a4_nvfp4_moe.py:317]: `when the Tensor Parallel degree is greater than the total number of experts`

Is that even possible..

**Comment by Jiminator** [comment]
I think this is fine as a quick hotfix, but it might be best to fix it at the source in compressed_tensors_w4a4_nvfp4_moe.py in process_weights_after_loading(). Is the expand operation necessary now that FI expects a tensor with numel = 1?

**Review by Fridge003 — APPROVED** [review]
